Week3/homework/step2-6.js

```diff
@@ -6,11 +6,15 @@ const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 function flattenArray2d(arr) {
   // Replace this comment and the next line with your code
   console.log(arr);
+  return arr.reduce((acc, element) => acc.concat(element), []);
 }
 
 function flattenArray3d(arr) {
   // Replace this comment and the next line with your code
   console.log(arr);
+  return arr
+    .reduce((acc, element) => acc.concat(element), [])
+    .reduce((acc, element) => acc.concat(element), []);
```

Comment:
Really good usage of `reduce` in this file, happy to see you use it, this is a good solution.